client/src/pages/ManageBookingPage/NextBookingComponent.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState, useEffect } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardHeader from '@material-ui/core/CardHeader';
+import CardContent from '@material-ui/core/CardContent';
+import Avatar from '@material-ui/core/Avatar';
+import IconButton from '@material-ui/core/IconButton';
+import Typography from '@material-ui/core/Typography';
+import { red } from '@material-ui/core/colors';
+import SettingsIcon from '@material-ui/icons/Settings';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: 400,
+    height: 180,
+  },
+  contents: {
+    padding: theme.spacing(1),
+  },
+  time: {
+    marginLeft: theme.spacing(1),
+    marginBottom: theme.spacing(1),
+  },
+  avatar: {
+    backgroundColor: red[500],
+    float: 'left',
+    width: theme.spacing(6),
+    height: theme.spacing(6),
+    marginLeft: theme.spacing(1),
+  },
+  name: {
+    float: 'left',
+    marginLeft: theme.spacing(1),
+    padding: theme.spacing(1.5),
+  },
+}));
+
+export default function NextBookingComponent() {
+  const classes = useStyles();
+
+  return (
+    <Card className={classes.root}>
+      <CardHeader
+        action={
+          <IconButton aria-label="settings">
+            <SettingsIcon />
+          </IconButton>
```

Comment:
So I know this button is in the mockups, but I'm not sure what we would actually use it for. Any ideas? If we can't think of anything I think you can remove it

Reply:
This button functions for modify booking which is I'm working on